src/main/java/org/skriptlang/skript/lang/comparator/InverseComparator.java

```diff
@@ -50,7 +50,7 @@ public boolean supportsInversion() {
 
 	@Override
 	public String toString() {
-		return "InverseComparator{" + comparator + "}";
+		return "InverseComparator{comparator=" + comparator + "}";
```

Comment:
In this case it's more for ConvertedComparator. I can see about making enhancements here though to address other types of Comparator

Reply:
I took another look at this, and I could make InverseComparator store ComparatorInfo rather than a Comparator itself. However, it would mean I need to create additional ComparatorInfos that are only used for toString. Do you think that's worth it?

Reply:
Yes, it's not any help seeing the object toString in a Spark performance monitor, I need to be able to see what it's doing.